packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -405,8 +423,9 @@ class _DropdownRoutePage<T> extends StatelessWidget {
     // than the button height and the button is at the very bottom or top of the
     // screen, the menu will be aligned with the bottom or top of the button
     // respectively.
-    if (menuTop < topLimit)
+    if (menuTop < topLimit) {
```

Comment:
Previous syntax was fine: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#separate-the-if-expression-from-its-statement